templates/webapi/test/infopanel.html.ep

```diff
@@ -12,14 +12,15 @@
                     </span>
                 % }
             </div>
-            <div id="info-box-content" class="card-body"
+            <div id="info-box-content" class="card-body row"
                 % if ($additional_data) {
                     data-favicon-url-16="<%= favicon_url '-16.png' %>" data-favicon-url-svg="<%= favicon_url '.svg' %>"
                 % }
             >
+                <div class="col-xl">
                 <div>
                     % if ($job->state eq DONE) {
-                        <div class="position-absolute top-right pt-2-and-half pr-3">
+                        <div class="position-absolute top-right badge-pill badge-light">
```

Comment:
keep in mind that `badge-pill` is dropped from BS5

Reply:
Ok. I guess we can very easily replace that with our own rules, though.